Makefile

```diff
@@ -1,10 +1,20 @@
 PROJECT := github.com/juju/utils
+GOFMT := $(shell gofmt -l .| sed -e "s/^/ /g")
```

Comment:
It seems a little odd to run this unconditionally. Is it possible to put this under check-go?